### PR TITLE
make install btn dynamic  in marketplace extension 

### PR DESCRIPTION
**Pull request by darwin808**
added fetchwrapper component in install btn
make marketplace install btn dynamic based on props 
created additinnal components appinstall,resource link and coming soon 
changes of the styles in /docs/

### CODE REVIEW

**Comment by darwin808** [comment]
image
change theme of accordion

File: src/components/Accordion/AccordionComp.js [hunk 1]
```diff
@@ -9,6 +9,11 @@ import * as Style from './styles';
 import { Box, Button, Link, useTheme } from '@mui/material';
 import { useRouter } from 'next/router';
 
+const customStyle = {
+  color: '#333333',
```
Comment (ardeay) [inline src/components/Accordion/AccordionComp.js:13]: use zesty color, theme.pallete.zesty.grey or theme.palete.zesty.darkblue

File: src/components/Accordion/styles.js [hunk 2]
```diff
@@ -69,3 +72,88 @@ export const CustomButton = styled.button`
...
+  justify-items:flex-start;
+  text-align: left;
+  outline: none;
+  font-size:14px;
+  font-weight: 100;
+  cursor: pointer;
+  border: 1px solid transparent;
+  // border-radius:5px;
+  transition: all 0.1s ease-in-out;
+  color: ${(props) => (props.active ? '#497edf' : '#333333')};
```
Comment (ardeay) [inline src/components/Accordion/styles.js:91]: use Zesty colors from theme

File: src/components/Accordion/styles.js [hunk 3]
```diff
@@ -69,3 +72,88 @@ export const CustomButton = styled.button`
...
+  // border-left: 1px solid #333333;
+
+
+  &:hover {
+    color: #497edf;
```
Comment (ardeay) [inline src/components/Accordion/styles.js:96]: zesty colors

File: src/components/Accordion/styles.js [hunk 4]
```diff
@@ -69,3 +72,88 @@ export const CustomButton = styled.button`
...
+    background: #fafafa;
```
Comment (ardeay) [inline src/components/Accordion/styles.js:97]: theme color paper level2

File: src/components/Accordion/styles.js [hunk 5]
```diff
@@ -69,3 +72,88 @@ export const CustomButton = styled.button`
...
+  }
+
+  &:focus {
+    font-weight: 800;
+    color: #497edf;
+  }
+  &:active {
+    color: #497edf;
```
Comment (ardeay) [inline src/components/Accordion/styles.js:105]: theme colors

File: src/components/Accordion/styles.js [hunk 6]
```diff
@@ -69,3 +72,88 @@ export const CustomButton = styled.button`
...
+
+
+  &:hover {
+    color: #497edf;
+    background: #fafafa;
+  }
+
+  &:focus {
+    font-weight: 800;
+    color: #497edf;
```
Comment (ardeay) [inline src/components/Accordion/styles.js:102]: theme colors

File: src/components/Accordion/styles.js [hunk 7]
```diff
@@ -69,3 +72,88 @@ export const CustomButton = styled.button`
...
+  &:active {
+    color: #497edf;
+    transform:scale(0.99);
+  }
+
+'`;
+
+export const AccordionBtnHead = styled.button`
+  width: 100%;
+  background: #fff;
```
Comment (ardeay) [inline src/components/Accordion/styles.js:113]: theme color background paper

File: src/components/Accordion/styles.js [hunk 8]
```diff
@@ -69,3 +72,88 @@ export const CustomButton = styled.button`
...
+  justify-items:flex-start;
+  text-align: left;
+  outline: none;
+  font-size:14px;
+  font-weight: bold;
+  cursor: pointer;
+  border: 1px solid transparent;
+  border-radius:5px;
+  transition: all 0.1s ease-in-out;
+  color: ${(props) => (props.active ? '#497edf' : '#333333')};
```
Comment (ardeay) [inline src/components/Accordion/styles.js:126]: theme

File: src/components/Accordion/styles.js [hunk 9]
```diff
@@ -69,3 +72,88 @@ export const CustomButton = styled.button`
...
+
+  &:hover {
+    color: #4973df;
```
Comment (ardeay) [inline src/components/Accordion/styles.js:129]: theme

File: src/components/Accordion/styles.js [hunk 10]
```diff
@@ -69,3 +72,88 @@ export const CustomButton = styled.button`
...
+  }
+
+  &:focus {
+    font-weight: 800;
+    color: #497edf;
```
Comment (ardeay) [inline src/components/Accordion/styles.js:134]: theme

File: src/components/Accordion/styles.js [hunk 11]
```diff
@@ -69,3 +72,88 @@ export const CustomButton = styled.button`
...
+'`;
+
+export const IconContainer = styled.button`
+  display: flex;
+  padding: 0.5rem 0;
+  align-items: center;
+  outline: none;
+  border: none;
+  height: 0.5rem;
+  background: #fff;
```
Comment (ardeay) [inline src/components/Accordion/styles.js:150]: theme

File: src/components/Accordion/styles.js [hunk 12]
```diff
@@ -69,3 +72,88 @@ export const CustomButton = styled.button`
...
+  &:hover {
+    color: #497edf;
```
Comment (ardeay) [inline src/components/Accordion/styles.js:152]: theme

File: src/components/Accordion/styles.js [hunk 13]
```diff
@@ -69,3 +72,88 @@ export const CustomButton = styled.button`
...
+  }
+
+  &:focus {
+    font-weight: 800;
+    color: #497edf;
```
Comment (ardeay) [inline src/components/Accordion/styles.js:157]: theme

File: src/views/marketplace/Extension.js [hunk 14]
```diff
@@ -80,6 +85,42 @@ const YoutubeEmbed = ({ youtubeHash }) => {
   );
 };
 
+const InstallButton = ({ data, theme }) => {
+  if (data.app_zuid) {
```
Comment (ardeay) [inline src/views/marketplace/Extension.js:89]: make this chain of IF statement an if else, then you dont need extra clauses. There will only ever be one component to render

**Review by ardeay — CHANGES_REQUESTED** [review]
For custom styles, always instantiate the theme and use theme colors.

**Review by ardeay — APPROVED** [review]

**Pull request merged**